<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">pyui-generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/exec.png</normaloff>:/icon/exec.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
    <item>
     <layout class="QVBoxLayout" name="layoutItems">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarFile">
   <property name="windowTitle">
    <string>project</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
   <addaction name="actionNew"/>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QToolBar" name="toolBarItem">
   <property name="windowTitle">
    <string>file</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddItem"/>
   <addaction name="actionAddMulti"/>
   <addaction name="actionGenerateAll"/>
   <addaction name="actionSync"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/exit.png</normaloff>:/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string notr="true">quit</string>
   </property>
   <property name="iconText">
    <string notr="true">quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/save.png</normaloff>:/icon/save.png</iconset>
   </property>
   <property name="text">
    <string notr="true">save</string>
   </property>
   <property name="iconText">
    <string notr="true">save</string>
   </property>
   <property name="toolTip">
    <string>Save project</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/load.png</normaloff>:/icon/load.png</iconset>
   </property>
   <property name="text">
    <string notr="true">load</string>
   </property>
   <property name="iconText">
    <string notr="true">load</string>
   </property>
   <property name="toolTip">
    <string>Load project</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/new.png</normaloff>:/icon/new.png</iconset>
   </property>
   <property name="text">
    <string notr="true">new</string>
   </property>
   <property name="iconText">
    <string notr="true">new</string>
   </property>
   <property name="toolTip">
    <string>New project</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+N</string>
   </property>
  </action>
  <action name="actionAddItem">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/add.png</normaloff>:/icon/add.png</iconset>
   </property>
   <property name="text">
    <string notr="true">addItem</string>
   </property>
   <property name="iconText">
    <string notr="true">addItem</string>
   </property>
   <property name="toolTip">
    <string>Add a file</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl++</string>
   </property>
  </action>
  <action name="actionGenerateAll">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/execAll</normaloff>:/icon/execAll</iconset>
   </property>
   <property name="text">
    <string notr="true">generateAll</string>
   </property>
   <property name="iconText">
    <string notr="true">generateAll</string>
   </property>
   <property name="toolTip">
    <string>Generate all files</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+G</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/autoSave.png</normaloff>:/icon/autoSave.png</iconset>
   </property>
   <property name="text">
    <string>sync</string>
   </property>
   <property name="iconText">
    <string notr="true">autoGenerate</string>
   </property>
   <property name="toolTip">
    <string>synchronize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAddMulti">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/addMulti.png</normaloff>:/icon/addMulti.png</iconset>
   </property>
   <property name="text">
    <string notr="true">addMulti</string>
   </property>
   <property name="iconText">
    <string notr="true">addMulti</string>
   </property>
   <property name="toolTip">
    <string>Add several files</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+M</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
